### DISCUSSION TITLE
Manually calculate the complexity in python

### DISCUSSION THREAD

**Discussion by AKHCE** [MilesCranmer/PySR#457] [Q&A]
Hi,
I am wondering if there is a built-in method in pysr by which users can calculate the complexity of equations. Or is there a way to calcualte complexity using sympy or other libs.

Any advice is much apprecited, 
Thanks

**Comment by MilesCranmer** [comment]
Maybe see
- https://github.com/MilesCranmer/PySR/discussions/380, or
- https://github.com/MilesCranmer/PySR/issues/339?

**Comment by AKHCE** [comment]
Thanks
